membership-attribute-service/conf/routes

```diff
@@ -15,14 +15,17 @@ OPTIONS    /user-attributes/me/membership-update-card       controllers.AccountC
 POST       /user-attributes/me/digitalpack-update-card      controllers.AccountController.digitalPackUpdateCard
 OPTIONS    /user-attributes/me/digitalpack-update-card      controllers.AccountController.digitalPackUpdateCard
 
-POST       /user-attributes/me/paper-update-card      controllers.AccountController.paperUpdateCard
-OPTIONS    /user-attributes/me/paper-update-card      controllers.AccountController.paperUpdateCard
+POST       /user-attributes/me/paper-update-card            controllers.AccountController.paperUpdateCard
+OPTIONS    /user-attributes/me/paper-update-card            controllers.AccountController.paperUpdateCard
```

Comment:
I would probably avoid adding removing spaces so the PR keeps simple

Reply:
this was just to have everything inline vertically